config/initializers/assets.rb

```diff
@@ -11,4 +11,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( admin.js admin.css )
```

Comment:
__[RuboCop]__

Layout/SpaceInsidePercentLiteralDelimiters: Do not use spaces inside percent literal delimiters.


<!-- Comment by [SideCI](https://sideci.com) -->